Python/pytime.c

```diff
@@ -1062,13 +1048,9 @@ py_win_perf_counter_frequency(_PyTimeFraction *base, int raise_exc)
     // * 10,000,000 (10 MHz): 100 ns resolution
     // * 3,579,545 Hz (3.6 MHz): 279 ns resolution
     if (_PyTimeFraction_Set(base, SEC_TO_NS, denom) < 0) {
-        if (raise_exc) {
-            PyErr_SetString(PyExc_RuntimeError,
-                            "invalid QueryPerformanceFrequency");
-        }
-        return -1;
+        return PyStatus_Error("invalid QueryPerformanceFrequency");
```

Comment:
Please use `_PyStatus_ERR()` instead: it stores the name of the current function, which gives more information in case of failure.

```suggestion
        return _PyStatus_ERR("invalid QueryPerformanceFrequency");
```

Same remark below (py_mach_timebase_info).